db/seeds.rb

```diff
@@ -136,3 +136,18 @@ def seed_childcare_providers!
 end
 
 Setting.create!(course_start_date: 3.months.from_now)
+
+# APIToken for testing
+if Rails.env.in?(%w[development review])
+  {
+    "Ambition Institute" => "ambition-token",
```

Comment:
maybe we can include tokens for all NPQ lead providers? and if Capita isn't there we can remove?